### PR TITLE
chore: Bump FastAdapter version and fix errors

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1752

### CODE REVIEW

File: app/src/main/java/com/eventyay/organizer/core/attendee/list/AttendeesFragment.java [hunk 1]
```diff
@@ -243,9 +244,9 @@ private void setupRecyclerView() {
             }
         );
 
-        fastAdapter = FastAdapter.with(Collections.singletonList(fastItemAdapter));
+        fastAdapter = FastAdapter.Companion.with(Collections.singletonList(fastItemAdapter));
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/attendee/list/AttendeesFragment.java]: Should work without Companion

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/attendee/list/AttendeesFragment.java]: Yes it's working. Updated.

File: app/src/main/java/com/eventyay/organizer/core/attendee/list/AttendeesFragment.java [hunk 2]
```diff
@@ -156,11 +157,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     private void sortAttendees(int sortBy) {
         if (sortBy == SORTBYTICKET) {
-            fastItemAdapter.withComparator((Attendee a1, Attendee a2) -> a1.getTicket().getType().compareTo(a2.getTicket().getType()));
+            ((ComparableItemListImpl<Attendee>) fastItemAdapter.getItemList()).withComparator((Attendee a1, Attendee a2) -> a1.getTicket().getType().compareTo(a2.getTicket().getType()), true);
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/attendee/list/AttendeesFragment.java]: Everything except this looks good. Find a way you don't need to cast

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/attendee/list/AttendeesFragment.java]: Couldn't find any other way. Check this: https://github.com/mikepenz/FastAdapter/issues/675#issuecomment-392546388

**Pull request merged**